galgebra/mv.py

```diff
@@ -2006,8 +2006,6 @@ def consolidate_coefs(self):
         new_coefs = []
         new_pdiffs = []
         for (coef, pd) in self.terms:
-            if isinstance(coef, Mv) and coef.is_scalar():
-                coef = coef.obj
```

Comment:
Other parts of this file assume that coeffs are always `Mv` instances. I don't see a good reason to mix-and-match scalars and `Mv`s here, given that `Sdop` is used for the cases when the coeffs are all scalars